core/linker/default.h

```diff
@@ -197,10 +197,17 @@ SECTIONS
         __uvisor_stack_start_boundary__ = .;
         . += STACK_GUARD_BAND;
 #endif
+#ifdef ARCH_MPU_KINETIS
+    . = ALIGN(32);
```

Comment:
Add comment why 32 bytes alignment is needed or use an informative define like KINETIS_REGION_ALIGNMENT

Reply:
comment added